### PR TITLE
Signup: update illustrations for NUX flow.

### PR DESCRIPTION
**Pull request by taggon**
This PR will replace the NUX flow icons with fancy illustrations.

See p5uIfZ-664-p2

*Note* This PR adds control flows and/or props to some shared components. They are necessary to isolate the changes from other flows until the new illustrations are used across the entire application.

### CODE REVIEW

File: client/my-sites/plan-features/header/tagline.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+	switch ( planType ) {
+		case PLAN_FREE:
+			tagline = translate( 'Not sure? Start here' );
+			break;
+		case PLAN_PERSONAL:
+			tagline = translate( 'Best for Bloggers' );
+			break;
+		case PLAN_PREMIUM:
+			tagline = translate( 'Best for Enterpreneurs & Freelancers' );
```
Comment (a8ci18n) [inline client/my-sites/plan-features/header/tagline.jsx]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Best for Entrepreneurs &amp; Freelancers' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/illustration-signup-flow)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/plan-features/header/tagline.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	PLAN_BUSINESS,
+	PLAN_PERSONAL
+} from 'lib/plans/constants';
+
+export function PlanFeaturesHeaderTagline( { planType, translate } ) {
+	let tagline;
+
+	switch ( planType ) {
+		case PLAN_FREE:
+			tagline = translate( 'Not sure? Start here' );
```
Comment (a8ci18n) [inline client/my-sites/plan-features/header/tagline.jsx]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Not sure? Start here.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by fditrapani** [comment]
Tested these out locally and they look great! Thanks @taggon

**Comment by fditrapani** [comment]
image

**Comment by fditrapani** [comment]
image

File: client/my-sites/plan-features/header/tagline.jsx (hunk 1 above)
Comment (taggon) [inline client/my-sites/plan-features/header/tagline.jsx]: 👍

**Comment by markryall** [comment]
This PR is as you say MASSIVE ;)

Everything does work as promised however.

Just wondering if you want to put all of these new svgs in separate components rather than inline.  They seem pretty large and have a significant impact on bundle sizes.  If so, perhaps these could be added as earlier commits so this isn't so unwieldy.

Perhaps this has nothing to do with your changes but I noticed that if refreshing the thank you page in dev mode (locally or on calypso.live) results in the random 'Skipping initial state rehydration to recreate first-load experience.', then the page will fail to render.  This is presumably because it does not trigger requests for the data it needs - it relies on that state already having been loaded.

**Comment by taggon** [comment]
@markryall Good catch! I just made them inline because the previous icons were so. I'll fix it.

**Comment by fditrapani** [comment]
Any updates on this @taggon?

**Comment by taggon** [comment]
After having a discussion, we've decided to skip the plans page until @fditrapani's signup test ends to avoid conflicts.

**Comment by fditrapani** [comment]
@taggon @cristelrossignol the plans signup test has been completed. You can go ahead and implement the illustrations here when you're ready.

**Comment by taggon** [comment]
Close as this PR was broke down into the five small PRs.

**Pull request closed without merging**